vsintegration/src/FSharp.Editor/CodeFixes/RemoveUnusedOpens.fs

```diff
@@ -21,30 +21,23 @@ type internal RemoveUnusedOpensCodeFixProvider [<ImportingConstructor>] () =
     override _.FixableDiagnosticIds =
         ImmutableArray.Create FSharpIDEDiagnosticIds.RemoveUnnecessaryImportsDiagnosticId
 
-    member this.GetChanges(document: Document, diagnostics: ImmutableArray<Diagnostic>, ct: CancellationToken) =
-        backgroundTask {
-            let! sourceText = document.GetTextAsync(ct)
-
-            let changes =
-                diagnostics
-                |> Seq.map (fun d ->
-                    sourceText
-                        .Lines
-                        .GetLineFromPosition(
-                            d.Location.SourceSpan.Start
-                        )
-                        .SpanIncludingLineBreak)
-                |> Seq.map (fun span -> TextChange(span, ""))
-
-            return changes
-        }
-
-    override this.RegisterCodeFixesAsync ctx : Task =
-        backgroundTask {
-            if ctx.Document.Project.IsFSharpCodeFixesUnusedOpensEnabled then
```

Comment:
This check is done in the analyzer itself hence we don't have to duplicate it. This is also how the other code fix in this PR works.

Proof:

https://github.com/dotnet/fsharp/assets/5451366/2d4111e0-9217-4c19-8ff5-09ed72aac651